EIPS/eip-615.md

```diff
@@ -1,27 +1,27 @@
 ---
 eip: 615
 title: Subroutines and Static Jumps for the EVM
-status: Draft
+status: Last Call
```

Comment:
Last call deadline field is missing.

Reply:
`review-period-end:` is missing indeed
According to EIP-1 that's normally 14 days, but I suppose it's up to the author.

Reply:
I think EIP-1 leaves the date to the discretion of the Editor, not the author.